javascript/javascript3/redo/week2/movies.js

```diff
@@ -0,0 +1,9 @@
+fetch('https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json')
+    .then(Response => Response.json())
```

Comment:
Nitpick(naming): Prefer to use camelCase for variables and functions, save PasalCase for classes and types.